item.rb

```diff
@@ -2,8 +2,8 @@ class Item
   attr_accessor :publish_date, :archived, :authors
   attr_reader :id
 
-  def initialize(publish_date, archived: false)
-    @id = Random.rand(1..1000) # Guys, modified id because we need to have an id generated here
+  def initialize(publish_date, archived: false, id: 0)
```

Comment:
@kifle23, I have not understood this change. Can you please explain why you made this change?

Reply:
> @kifle23, I have not understood this change. Can you please explain why you made this change?

Hey, @Nasirkhan294 thank you for your review, this default parameter change is needed because we needed to read data/add data from/to the JSON file. It helps us to:
1. Avoid duplicate entries
2. Reading data by id if this is required in the future.

I would like to say thank you again for your review 🙏